.gitignore

```diff
@@ -14,6 +14,7 @@
 *.pyc
 _build
 .vscode
+.idea
```

Comment:
Is this necessary?

Reply:
I used CLion to make these changes locally. JetBrains IDEs (CLion, IntelliJ) are pretty popular so it's probably worth including in the .gitignore file.